recipe/bld.bat

```diff
@@ -22,22 +22,19 @@ set "PYTHONIOENCODING=UTF-8"
 @REM See hardcoded-paths.patch
 set "CPPFLAGS=%CPPFLAGS% -D^"%LIBRARY_PREFIX_M%^""
 
-%PYTHON% %BUILD_PREFIX%\Scripts\meson -h
-
-%BUILD_PREFIX%\python.exe %BUILD_PREFIX%\Scripts\meson --buildtype=release --prefix="%LIBRARY_PREFIX_M%" --backend=ninja -Diconv=external -Dselinux=disabled -Dxattr=false -Dlibmount=disabled ..
-if errorlevel 1 exit 1
+meson setup --buildtype=release --prefix="%LIBRARY_PREFIX_M%" --backend=ninja -Dselinux=disabled -Dxattr=false -Dlibmount=disabled ..
+if errorlevel 1 (
+  type forgebuild\meson-logs\meson-log.txt
+  exit /b 1
+)
 
 ninja -v
 if errorlevel 1 exit 1
 
-@REM Lots of tests fail right now
-@REM ninja test
-@REM if errorlevel 1 exit 1
-
-ninja install
-if errorlevel 1 exit 1
-
-del %LIBRARY_PREFIX%\bin\*.pdb
-
-@REM For some reason conda-build decides that the meson files in Scripts are new?
-del %PREFIX%\Scripts\meson* %PREFIX%\Scripts\wraptool*
```

Comment:
this got taken out - are these files now being packaged?

Reply:
These are meson scripts. Whatever the issue was, it doesn't seem to apply anymore. At least I don't find those in the resulting packages.

Reply:
thanks for checking